spyder/utils/pixi.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""Conda/anaconda utilities."""
+
+# Standard library imports
+from functools import lru_cache
+import os
+
+# Third-party imports
+from packaging.version import parse
+
+# Local imports
+from spyder.utils.programs import find_program, run_program
+
+WINDOWS = os.name == "nt"
+
+
+def find_pixi(pyexec=None):
```

Comment:
Given that pixi is used to only handle conda envs, I think we should move this function to `spyder/utils/conda.py` instead. In addition, I think we don't need an additional `conda_version` just for pixi (see below).